src/stacked_borrows.rs

```diff
@@ -338,16 +338,20 @@ impl<'tcx> Stack {
         &mut self,
         access: AccessKind,
         tag: SbTag,
-        dbg_ptr: Pointer<AllocId>, // just for debug printing amd error messages
+        (alloc_id, range): (AllocId, AllocRange), // just for debug printing amd error messages
         global: &GlobalState,
     ) -> InterpResult<'tcx> {
         // Two main steps: Find granting item, remove incompatible items above.
 
         // Step 1: Find granting item.
         let granting_idx = self.find_granting(access, tag).ok_or_else(|| {
             err_sb_ub(format!(
-                "no item granting {} to tag {:?} at {:?} found in borrow stack.",
-                access, tag, dbg_ptr,
+                "no item granting {} to tag {:?} over {}[{:#x}..{:#x}] found in borrow stack.",
```

Comment:
This still says "borrow stack" so I guess that alleviates my concern about mentioning that term? I am a bit confused now.^^

Reply:
Though this makes me wonder if this should also be reworded to be consistent with your new reborrowing errors?

Reply:
I did my best to reword where I could in response to your comment about mentioning borrow stacks.

The code missing the borrow stack terminology is in `grant_error` now.

Reply:
I think I see what you mean.

I guess I came around to not having as strong an opinion about this as I used to. If you want to instead word things like "but parent tag {} is invalid for this memory range" (which seems more consistent with "parent tag {:?} grants insufficient permission over this memory range"), that is fine for me.